glib/src/gobject/auto/signal_group.rs

```diff
@@ -0,0 +1,76 @@
+// This file was generated by gir (https://github.com/gtk-rs/gir)
+// from gir-files (https://github.com/gtk-rs/gir-files)
+// DO NOT EDIT
+
+use crate::object::IsA;
+use crate::object::ObjectType as ObjectType_;
```

Comment:
This is unused and is causing a warning. Is there a way to suppress this warning, or stop this statement from being generated?

Reply:
Can you create an issue about that? It's a problem in gir. In other places there's already an `#[allow(unused_imports)]` on top of `mod auto` in `lib.rs` for the very same reason.